packages/selenium/test/RunPartial/IncludeExcludeTests.cs

```diff
@@ -0,0 +1,40 @@
+using System;
+using Deque.AxeCore.Selenium;
+using NUnit.Framework;
+
+namespace Deque.AxeCore.Selenium.Test.RunPartial
+{
+    public class IncludeExcludeTests : TestBase
```

Comment:
These ought to be a little more comprehensive than "doesn't throw" - it would be good to actually verify that they're including/excluding results from the elements in question as expected